LoopUI/Charts/PredictedGlucoseChart.swift

```diff
@@ -299,4 +299,14 @@ extension PredictedGlucoseChart {
             }
         }
     }
+    
+    public struct PredictedGlucoseBounds {
+        var minimum: HKQuantity
+        var maximum: HKQuantity
+        
+        public static var `default`: PredictedGlucoseBounds {
+            return PredictedGlucoseBounds(minimum: HKQuantity(unit: .millimolesPerLiter, doubleValue: 40),
```

Comment:
This doesn't look right.  40-400 is mg/dL, not mmol/L.

Reply:
yikes! good catch.